File: exc4_Vetor.c
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main()
{
    int i, vet[15], c;

    setlocale(LC_ALL, "Portuguese");

    for(i = 0; i <= 14; i++){
        printf("Informe o número %d: ", i);
        scanf("%d", &vet[i]);
    }

    c = 0;

    for(i = 0; i <= 14; i++){
        if (vet[i] == 30){
            printf("\nA posição igual a 30 é: %d", i);
            c = c + 1;
        }

    }

    printf("\nA quantidade de números iguais a 30 é: %d", c);


    return 0;
}

File: exc4_Vetor.rb
vet=[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]

for l in 0..14
        puts "digite os valores do no vetor[#{l}]"
        vet [l] = gets.chomp.to_i
        puts vet [l]
    
end

c = 0

for l in 0..14
    if (vet [l] == 30)
        puts "A posição igual a 30 é: #{l}"
        c= c + 1
    end
end

puts "A quantidade de números iguais a 30 é: #{c}"